crates/type-system/tsconfig.json

```diff
@@ -1,3 +1,4 @@
 {
-  "extends": "@local/tsconfig/node.json"
+  "extends": "@local/tsconfig/node.json",
+  "exclude": ["./target/*"]
```

Comment:
At what point do we get files in `target`? We can delete any temp files as part of the build.

Reply:
Target is helpful for caching, it's basically the analogue of `.next` or `dist`

Reply:
Deleting it would not be optimal (due to losing the caching benefits)

Reply:
I see, let's keep it as is then.